src/Shared/HandyControl_Shared/Controls/Cover/CoverFlow/CoverFlow.cs

```diff
@@ -186,6 +186,21 @@ public void AddRange(IEnumerable<object> contentList)
         /// </summary>
         public void JumpTo(int index) => _jumpToIndex = index;
 
+        public void Next()
+        {
+            PageIndex = PageIndex + 1;
+        }
+
+        public void Previous()
+        {
+            PageIndex = PageIndex - 1;
```

Comment:
Will negative numbers be calculated here?

Reply:
Everything works fine
I do not think there is a need for this 

![Animation](https://user-images.githubusercontent.com/9213496/131877071-3b20a022-06d0-4952-917a-2f13df1f84a1.gif)